packages/block-library/src/cover/edit/block-controls.js

```diff
@@ -73,25 +75,34 @@ export default function CoverBlockControls( {
 		} );
 	};
 
+	const mode = useSelect(
+		( select ) => select( blockEditorStore ).__unstableGetEditorMode(),
+		[]
+	);
+
 	return (
 		<>
-			<BlockControls group="block">
-				<BlockAlignmentMatrixControl
-					label={ __( 'Change content position' ) }
-					value={ contentPosition }
-					onChange={ ( nextPosition ) =>
-						setAttributes( {
-							contentPosition: nextPosition,
-						} )
-					}
-					isDisabled={ ! hasInnerBlocks }
-				/>
-				<FullHeightAlignmentControl
-					isActive={ isMinFullHeight }
-					onToggle={ toggleMinFullHeight }
-					isDisabled={ ! hasInnerBlocks }
-				/>
-			</BlockControls>
+			{ 'navigation' !== mode && (
```

Comment:
I think we should do this the same way that the image block does, like here: https://github.com/WordPress/gutenberg/blob/f1a4f1cef8e17262e976c282bb5bc2c7591e5549/packages/block-library/src/image/image.js#L334

Reply:
@richtabor updated as per your suggestion, please review.